### ISSUE TITLE
Low sonar-scanning cache hit ratio in Travis CI

### ISSUE THREAD

**Issue by abitmore** [bitshares/bitshares-core#2292]
**Build Error Description**

Recently, Travis CI often fails due to timeout on SonarQube scanning, probably cause by some changes in latest sonar-scanner. It turns out that the cache hit ratio when scanning became relatively low.

For example, here is a typical develop build: https://travis-ci.org/github/bitshares/bitshares-core/builds/735479596.
* In the 4th job (scan with sonar, step 2), https://travis-ci.org/github/bitshares/bitshares-core/jobs/735479606, there is
  ```
  INFO: Cache: 57/57 hits, 30221139 bytes
  ...
  INFO: Cache: 8/105 hits, 49766903 bytes
  ```
  which means only 8 files out of the 57 files cached in the first pass was effective in the second pass.

* And in the 5th job and the 6th job
  ```
  INFO: Cache: 105/105 hits, 49766903 bytes
  ...
  INFO: Cache: 64/139 hits, 68053030 bytes
  ```
  ```
  INFO: Cache: 139/139 hits, 68053030 bytes
  ...
  INFO: Cache: 79/140 hits, 68929898 bytes
  ```

**Build Environment**

Travis CI. Ubuntu 16.04.

## CORE TEAM TASK LIST
- [ ] Evaluate `Build Error`
- [ ] Provide build guidance
- [ ] <OR> Create `Bug Report`

**Comment by abitmore** [comment]
Probably caused by https://jira.sonarsource.com/browse/CPP-2611 .

**Comment by abitmore** [comment]
Fixed by #2294.
